src/main/resources/db/migration/V001__Create_schema.sql

```diff
@@ -0,0 +1,10 @@
+-- *** Product catalog ***
+CREATE TABLE Product
+(
+  id BIGSERIAL,
+  modificationCounter INTEGER NOT NULL,
+  title VARCHAR (255),
```

Comment:
I think it would be better to mark the title (and maybe the price) with `NOT NULL` as well, because a product without title (and price) does not make sense.